packages/react-reconciler/src/ReactFiberHiddenContext.old.js

```diff
@@ -22,6 +22,7 @@ type HiddenContext = {
   // order to reveal the hidden content.
   // TODO: Remove `subtreeLanes` context from work loop in favor of this one.
   baseLanes: number,
+  ...
```

Comment:
`OffscreenState` has an extra property here (`cachePool`) and needs to be assignable somewhere.

Reply:
It should probably just add this field: https://github.com/facebook/react/blob/4ea064eb0915b355b584bff376e90dbae0e8b169/packages/react-reconciler/src/ReactFiberOffscreenComponent.js#L36